PendingReleaseNotes.md

```diff
@@ -2,18 +2,32 @@
 
 ## Action Required
 
-## Known Issues
-
-## Deprecations
-
 ## Notable Features
 
+- Rook Flexvolume
+  - Introduces a new Rook plugin based on [Flexvolume](https://github.com/kubernetes/community/blob/master/contributors/devel/flexvolume.md)
+  - Integrates with Kubernetes' Volume Controller framework
+  - Communicates with the Rook-agent via unix sockets
```

Comment:
the unix sockets part feels more like an implementation detail and not a user facing feature we want to share

Reply:
ok i can remove it